netci.groovy

```diff
@@ -1842,7 +1863,16 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                 triggerString += "\\W+Innerloop"
             }
             else {
-                contextString += " ${scenario}"
+                // for consistency with naming elsewhere:
```

Comment:
Please don't make this change. The benefit of the way it is currently done is you can copy/paste the display name of the job and use it as the trigger string. That is a very useful feature. Otherwise, if a job fails, you need to use "dotnet... help" to figure out all the jobs, and back map from "display name" to the trigger phrase. If GitHub would display both the trigger phrase AND a "pretty" display name, then it wouldn't be an issue.

IMO, until that happens, I'd love to change ALL the display names to be more like ARM is here right now.

Reply:
👍